array-manipulation-3/src/unique.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-unused-vars */
+function unique(array) {
+  var newArray = [];
+  if (array.length !== 0) {
+    newArray.push(array[0]);
+    for (var i = 1; i < array.length; i++) {
+      if (!newArray.includes(array[i])) {
+        newArray.push(array[i]);
+      }
+    }
+    return newArray;
+  }
```

Comment:
🤔 you should be able to loop through the input array from start to finish and leave our your conditional on line 4.